management/src/main/java/org/ehcache/management/registry/DefaultCollectorService.java

```diff
@@ -113,6 +121,20 @@ public void cacheRemoved(String alias, Cache<?, ?> cache) {
         EhcacheNotification.CACHE_REMOVED.name()));
   }
 
+  private void cacheCleared(String alias) {
+    collector.onNotification(
```

Comment:
Whatever this method does it's synchronous with the clear.  It's not anything expensive is it?

Reply:
nope; it's pushing an event